tools/build/Dockerfile

```diff
@@ -33,4 +33,9 @@ ENV CI=true
 
 COPY ./setup_linux_android.sh ./setup_linux_android.sh
 
+# We define the variable here, so it can be seen by both the setup script, and the running container
+ENV ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT:-"/opt/android-sdk"}
```

Comment:
Without this, when we finally run the build, it doesn't know where to find the SDK.